scripts/rtma3d/exrtma3d_gsianl.ksh

```diff
@@ -380,8 +464,28 @@ if [ "${envir}" == "lsf" ] || [ "${envir}" == "pbspro" ]; then #WCOSS
   fi
   echo "HVC option is $hybridcord"
 fi
+
+#====  set GSI namelist options for analysis of HOWV and GUST  ====#
+  oerr_gust="-1.0"                #Obs Err of gust (if<0, use preset value 1.0 defined in read_prepbufr.f90)
```

Comment:
Should this value be negative?

Reply:
@MatthewMorris-NOAA,
Hi Matthew,

if set it be a negative value, that means GSI would use its default value. I understand your concern that a negative value here is really confusing. Maybe I'd better to remove it in the namelist, so GSI still runs with the default value, and users would be not be confused with the negative value.

thanks,
-Gang